matplotlib2tikz/save.py

```diff
@@ -309,6 +309,8 @@ def _recurse(data, obj):
             content.extend(cont, child.get_zorder())
         elif isinstance(child, mpl.legend.Legend):
             data = legend.draw_legend(data, child)
+            if data['legend colors']:
+                content.extend(data['legend colors'], 0)
```

Comment:
I'm not an expert, but I think it has something to do with ContextManager. I saw this in save.py in line 281 and just used it in the same way.